### ISSUE TITLE
545 biographical or historical data

### ISSUE THREAD

**Issue by CECSpecialistI** [uwlib-cams/MARC2RDA#218]
https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/5XX.csv

**Comment by cwarc** [comment]
https://docs.google.com/spreadsheets/d/1lQo5_l4UMjN_lyoUACfEXAzm-NDUCezqXoTXXjUpJig/edit#gid=508627139

Hello Everyone,

Please see the first pass for Field Code 545 - Biographical or Historical Data (R). Is it possible someone might review this work?

Best,
Alice

**Comment by lake44me** [comment]
Looks as good as it can be.

**Comment by cspayne** [comment]
Hi @MHill-BL and @CECSpecialistI! I've reopened this because the code has some issues that need to be fixed.

Looking at the excel sheet, I have a question - we've got a condition where if a 130 is present in the record, this field maps to a note on expression with the transform note "attach note to entity minted by the main entry (1XX) of the MARC record". 

Isn't the 130 for a work? Do you mean to attach this as a note on work to the main work in the record? Or the main expression? We don't mint any additional entities based on a 130 as far as I am aware.

**Comment by tmqdeborah** [comment]
Looking at some sample records from LC (attached), I have to say that I don't think that we can reliably determine who or what this 545 note is for. 

[545 examples_LC50th.txt](https://github.com/user-attachments/files/20357867/545.examples_LC50th.txt)

My advice would be to simply map it as a note on manifestation generating the prefixes listed in the Transformation notes (based on the indicator values. That is, rows 11, 15, and 16 only.

**Comment by CECSpecialistI** [comment]
130 is the expression not necessarily the work. it should map to the expression entity in the main WEMI stack created from the MARC record.

**Comment by AdamSchiff** [comment]
130 can be an access point for a work or for an expression.  Just depends on what subfields are present (and what the intention of the cataloger was, and what set of rules were being used, and ....).

But generally speaking, a 130 with only $a, $n, and/or $p would represent a work.  A 130 with $l, $f, $h, and/or $s would represent an expression.  Not foolproof though, alas.  $f was used under AACR2 (maybe pre-AACR2 too) for some work access points, which would now not be separately subfielded in RDA.

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________

**Comment by CECSpecialistI** [comment]
Yeah I think this is why in our conversations we decided that expression was the safest bet, since the expression inherits the work attributes.

**Comment by AdamSchiff** [comment]
It's really a note on agent related to the work described by the metadata, usually the creator of the work.  Could it be given as a note on agent (or note on person, note on corporate body, etc.) in the metadata description of that agent instead of the manifestation?  Or note on work?  The 545 in bib records is basically equivalent to the 678 in authorities.

Interestingly, if you search in RDA Toolkit for "545" you only get one result with a mapping of element "script" to "MARC 21 Bibliographic 545 ** $6 script identification code [identifier] (field-level provenance) (data provenance)".  Otherwise, they haven't mapped 545 to anything in RDA!

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________

**Comment by MHill-BL** [comment]
Hi All, 
I'm not sure if what I’ve done is correct, so please do correct me if I’ve misunderstood anything, but based on the discussion, I’ve updated the transform so that 545 is only mapped as a note on the manifestation? Specifically I have: 

•	I removed the previous mappings that treated the 545 as a note on agent (rdaa:P50391, rdaa:P50392) or on expression (rdaa:P20071).
•	I now map 545 using only (rdam:P30137)

Thanks all, happy to tweak this further if needed.

Thank you 
Matt

**Comment by CECSpecialistI** [comment]
This seems ok to me, I think 545 is fine as a note on manifestation.

**Comment by tmqdeborah** [comment]
@MHill-BL Did you remember to generate the prefixes listed in the Transformation notes (based on the indicator 1 values):
# = Biographical or historical note:
0 = Biographical sketch: 
1 = Administrative history: 

I looked at [02895c6](https://github.com/uwlib-cams/MARC2RDA/commit/02895c66892fae86159e5ab11b39c7af41c5271d), but couldn't quite tell from what I saw.

**Comment by cspayne** [comment]
Hi @tmqdeborah, you can see the code for this in the [545 named template](https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/transformationCode/m2r-5xx-named.xsl#L729).

@MHill-BL the code looks good, thanks for working on this! The only thing that needs an update is that the prefix for the property should be rdamd: instead of rdam:

[This decision](https://github.com/uwlib-cams/MARC2RDA/wiki/Decisions-Index#iim3-datatypeobject-properties) is kind of buried, but when an object IRI is expected, we use the object property (so in this case it would be rdamo:), otherwise when a text value is expected we use the datatype property (rdamd:).

**Comment by cspayne** [comment]
Hi @MHill-BL, I think you're right, spacing looks good! I'm not sure why I put that, so apologies for any confusion.
